pkg/interceptors/github/github.go

```diff
@@ -0,0 +1,61 @@
+package github
+
+import (
+	"errors"
+	gh "github.com/google/go-github/github"
+	"net/http"
+
+	"github.com/tektoncd/triggers/pkg/interceptors"
+
+	triggersv1 "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+type Interceptor struct {
+	KubeClientSet kubernetes.Interface
+	Logger        *zap.SugaredLogger
+	Github        *triggersv1.GithubInterceptor
+}
+
+func NewInterceptor(ei *triggersv1.EventInterceptor, k kubernetes.Interface, l *zap.SugaredLogger) interceptors.Interceptor {
+	return &Interceptor{
+		Logger:        l,
+		Github:        ei.Github,
+		KubeClientSet: k,
```

Comment:
Maybe this should accept `GithubInterceptor` rather than `EventInterceptor`, to reduce the coupling, what do you think?

(Same for the WebhookInterceptor)

Reply:
Ah, done.